content/id/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -545,6 +545,9 @@ kubectl create -f https://kubernetes.io/examples/pods/pod-configmap-env-var-valu
 ```
 
 menghasilkan keluaran pada kontainer `test-container` seperti berikut:
+```shell
+kubectl logs dapi-test-pod
+```
```

Comment:
Hi @Arhell … why not just merged these two shells? because I believe this is command and it’s output.

Reply:
I don't mind, I just suggest to other PRs, this one is only synchronization with the English version